doc/source/security.md

```diff
@@ -155,3 +155,44 @@ This is a sensitive security issue (similar to writing sudo rules in a
 traditional computing environment), so be very careful.
 
 There's ongoing work on making this easier!
+
+
+## Kubernetes Network Policies
+
+Kubernetes has optional support for [network policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/) which let you restrict how pods can communication with each other and the outside world. This can provide additional security within JupyterHub, and can also be used to limit network access for users of JupyterHub.
+
+Your Kubernetes cluster may have partial, full, or no support for network policies, and Kubernetes will **silently ignore** policies that aren't supported. If you rely on a policy to restrict what users can access you should verify the policies are enforced on your cluster.
+
+Network policies are disabled by default, you can enable them in your `config.yaml`:
+
+```yaml
+hub:
+  networkPolicy:
+    enabled: true
+proxy:
+  networkPolicy:
+    enabled: true
+singleuser:
+  networkPolicy:
+    enabled: true
+```
+
+The default singleuser policy allows all outbound network traffic. To restrict outbound traffic to DNS, HTTP and HTTPS:
+
+```yaml
+singleuser:
+  networkPolicy:
+    enabled: true
+    egress:
+    - ports:
+      - port: 53
+        protocol: UDP
+    - ports:
+      - port: 80
+        protocol: TCP
+    - ports:
+      - port: 433
+        protocol: TCP
+```
+
+See the Kubernetes documentation for further information on defining policies.
```

Comment:
Perhaps provide a link here as well. This URL is the most relevant documentation on k8s right?
- https://kubernetes.io/docs/concepts/services-networking/network-policies/

Reply:
@manics I forgot to communicate a big warm appreciation for your contribution !! Thank you =D

Reply:
I'd already included that link in the first paragraph, but I guess there's no harm in repeating it.